tests/app/utils.ts

```diff
@@ -17,6 +17,9 @@ const assertNumCities = async (
   const numCities = await page.locator(CITY_MARKER).count();
   expect(numCities).toBeGreaterThanOrEqual(range[0]);
   expect(numCities).toBeLessThanOrEqual(range[1]);
+  // Checks for accurate city counter
+  const numCounter = await page.locator("#counter-numerator").innerText();
+  expect(numCities).toEqual(parseInt(numCounter.replace(/^\D+/g, ""), 10));
```

Comment:
Why should the number cities be == 10? It should be in the `range`, right?

Reply:
10 is the radix parameter for parseInt(), which I believe eslint wants to be included.

Reply:
Ah, you're totally right. My bad.